src/app/shared/components/hmcts-annotation-ui-lib/components/comments/comment-item/comment-item.component.html

```diff
@@ -16,6 +16,7 @@
   <textarea 
     #commentArea
     (blur)="onBlur()"
+    (click)="resize($event)"
```

Comment:
We can move this into the handleShowBtn() function.
`
    handleShowBtn() {
        this.focused = true;
        this.hideButton = false;
        this.resize();
    }`